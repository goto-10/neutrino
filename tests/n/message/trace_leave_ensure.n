def $leave_plain_ensure() => try null ensure leave.outer();
do $leave_plain_ensure();
