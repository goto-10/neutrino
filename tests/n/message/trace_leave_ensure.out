Error: %<condition: UncaughtSignal(escape)>
Info: --- backtrace ---
- leave.outer()
- ensure
- :leave_plain_ensure()
